electrolytes/runsimulations.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash 
+
+num_lines=$(wc -l < elytes.csv)
+num_lines=$((num_lines-1))
+
+for ((i = 0; i < num_lines; i++)); do
+    #Generate the system
+    cd $i
+    cp ../runsystem.py ./
+    python runsystem.py
+    cd ..
+done
```

Comment:
Do we want/need to run this in parallel?

Reply:
We can run them in parallel. Depending on how jobs are being submitted in the cluster, we could go GNU parallel as well.